wg-naming/README.md

```diff
@@ -0,0 +1,70 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+### Goals
+
+- Evaluate language and naming choices within the Kubernetes organization, with
+  a specific initial focus of:
+  - Removing language which cannot be fully separated from racist, sexist or
+    homophobic connotations in cultural contexts outside of computer
+    programming
+  - Improving clarity of codebases and documentation by replacing idioms,
+    metaphors and slang specific to the English language
+- Create a list of terms to avoid and propose alternatives
+- Define a process by which any member of the Kubernetes organization can
+  propose a language recommendation, have them evaluated based on a rubric, and
```

Comment:
Want to make sure I understand what "language recommendation" means here. Would it be something like "I'd like to recommend the project not to use XXX term because of YYY connotation"?

Reply:
Along those lines, yes. 

The idea with this bullet point is to accommodate the future with a process, so that the next time we're in a cultural moment like this one, we won't need to go through the WG formation process again. It'll essentially be documenting the how-to of proposing a change, and how any stakeholders involved in the future should evaluate that request.

Reply:
I suggest instead of "evaluated based on a rubric" you add another line item about defining a process for evaluating proposals. So, a process for suggesting some language is harmful, and a process for evaluating suggestions. Then the specifics of those processes don't need to block getting this PR merged.

(I'm not claiming a rubric is the wrong way to go, just that there's probably a discussion to be had and it doesn't need to happen here.)

Reply:
(sorry for poor conversational flow, github hadn't shown me the previous comment)

Reply:
Taking these suggestions and implementing per https://github.com/kubernetes/community/pull/4884/files#r445263740